alembic/script/base.py

```diff
@@ -782,6 +782,8 @@ def _rev_path(
         message: Optional[str],
         create_date: "datetime.datetime",
     ) -> str:
+        epoch_reference_date = datetime.datetime(1970, 1, 1)
+        epoch = int((create_date - epoch_reference_date).total_seconds())
```

Comment:
I'd rather not hardcode this here.  you can call `create_date.timestamp()`:

```py
>>> import time
>>> import datetime
>>> create_date = datetime.datetime.now()
>>> epoch_reference_date = datetime.datetime(1970, 1, 1)
>>> epoch = int((create_date - epoch_reference_date).total_seconds())
>>> epoch
1651522726
>>> create_date.timestamp()
1651537126.489675

```

Reply:
I think the latter is using UTC which accounts for the discrepancy

Reply:
Sure, the reason I chose that path was because going with `timestamp()` would need a `round(..., 0)` and the conversion to `int` to avoid the decimals, but I guess it should work without problems.

Do you think this `int(round(create_date.timestamp()))` could work?

```python
>>> import time
>>> import datetime
>>> create_date = datetime.datetime.now()
>>> epoch_reference_date = datetime.datetime(1970, 1, 1)
>>> epoch = int((create_date - epoch_reference_date).total_seconds())
>>> epoch
1651527473
>>> create_date.timestamp()
1651538273.443539
>>> int(round(create_date.timestamp(),0))
1651538273
>>>
```

Reply:
yeah just int it, i think that's fine

Reply:
great, just pushed the new version and fixes tests, everything seems to pass.
thanks a lot for the fast review @zzzeek